osidb/models.py

```diff
@@ -2092,6 +2092,10 @@ def is_community(self) -> bool:
     #      - see e.g. backends/jira/rest/request.py in SFM2
     # TODO unlike SFM2, SLA can't (?) evaluate the new streams part as long as this is a property
     #      on affect (since affect doesn't have streams). Should we file a new Jira to fix it?
+    # TODO: There's probably an omission in not creating a compliance-priority tracker label IN OSIDB.
```

Comment:
@osoukup There's probably an omission in not creating a compliance-priority tracker label IN OSIDB. See backends/jira/rest/request.py in SFM2 and nonexistence of adding the label in OSIDB. Am I misunderstanding something or shall we file a Jira for OSIDB?

Reply:
Yes. Wrote a comment about that to the Jira.